swaybar/render.c

```diff
@@ -411,7 +412,7 @@ static uint32_t render_workspace_button(cairo_t *cairo,
 	hotspot->height = height;
 	hotspot->callback = workspace_hotspot_callback;
 	hotspot->destroy = free;
-	hotspot->data = strdup(name);
+	hotspot->data = strdup(whole_name);
```

Comment:
No need for an additional variable here right? We could just `strdup(ws->name)`?

Reply:
I think you're missing the point of the patch here

Reply:
Shit I see what you mean

Reply:
@SirCmpwn you're too fast :)
@emersion It's such a small change that I don't think it really matters..

Reply:
I pushed a fixup commit